### PR TITLE
Tk 1

### PR DESCRIPTION
**Pull request by lvgarces**

### CODE REVIEW

File: src/Pokeitem/Pokeitem.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React, { useState } from "react";
```
Comment (davidlopez9926) [inline src/Pokeitem/Pokeitem.jsx]: delete useState, you are not using it

**Review by davidlopez9926 — COMMENTED** [review]
fix

**Review by davidlopez9926 — COMMENTED** [review]
aaaa

**Review by davidlopez9926 — CHANGES_REQUESTED** [review]
:(

File: src/Pokeitem/components/InfoPokemon.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import '../Pokeitem.scss';
+
+
+const InfoPokemon = ({no, name}) => {
+    return (
+        <div className="info-content">
+            <p className="info-poke">No. {no}</p>
+            <p className="info-poke">{name}</p>
```
Comment (davidlopez9926) [inline src/Pokeitem/components/InfoPokemon.jsx]: what if I put a large name? it should be cropped

File: src/Pokeitem/Pokeitem.jsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+import './Pokeitem.scss';
+import InfoPoke from './components/InfoPokemon'
+import img1 from "./MiniPokemon.png";
```
Comment (davidlopez9926) [inline src/Pokeitem/Pokeitem.jsx]: use more descriptive names for these images

File: src/Pokeitem/Pokeitem.jsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+import img2 from "./type.png";
+
+
+const Pokeitem = () => {
+    
+  return(
+    <label>
```
Comment (davidlopez9926) [inline src/Pokeitem/Pokeitem.jsx:11]: why a label and not a div?

File: src/Pokeitem/components/InfoPokemon.jsx [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
```
Comment (davidlopez9926) [inline src/Pokeitem/components/InfoPokemon.jsx]: Put this component in a InfoPokemon folder, with its own stylesheet

File: src/Pokeitem/Pokeitem.scss [hunk 6]
```diff
@@ -0,0 +1,61 @@
+.pokeitem {
+	display: none;
+	&:checked {
```
Comment (davidlopez9926) [inline src/Pokeitem/Pokeitem.scss]: you could simplfy this selector using
&:checked + .container-pokeitem instead

File: src/Pokeitem/Pokeitem.scss [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+		+ {
+			.container-pokeitem {
+				border: #FF6F6F;
```
Comment (davidlopez9926) [inline src/Pokeitem/Pokeitem.scss]: this could be a variable, since you are using this color twice

File: src/Pokeitem/Pokeitem.scss [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+				border-style: solid;
+				border-width: 0.5rem 0 0.5rem 0.5rem;
+				background: #FF6F6F;
+				box-shadow: 5px 1px 5px #0000000d, inset -5px -4px 6px #fc9b9b4d, inset 5px 4px 10px #00000026;
+				stroke: rgb(0, 92, 173);
```
Comment (davidlopez9926) [inline src/Pokeitem/Pokeitem.scss]: delete

File: src/Pokeitem/Pokeitem.scss [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+				stroke-width: 5rem;
```
Comment (davidlopez9926) [inline src/Pokeitem/Pokeitem.scss]: delete

File: src/Pokeitem/Pokeitem.scss [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+    display: flex;
+    flex-direction: row;
+    justify-content: start;
+
+    &:hover{
+        cursor: pointer;
+    }
+}
+
+.info-content{
```
Comment (davidlopez9926) [inline src/Pokeitem/Pokeitem.scss]: Put InfoPokemon styles in its own stylesheet

File: src/Pokeitem/Pokeitem.scss [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+    height: 5%;
+    margin-left: 1.5rem;
+    
+}
+.img-type{
+    width:1.3rem;
+    height: 1.3rem;
+    
+}
+.margin-img{
```
Comment (davidlopez9926) [inline src/Pokeitem/Pokeitem.scss]: this could be a mixin, used in both previous classes

File: src/Pokeitem/Pokeitem.scss [hunk 12]
```diff
@@ -0,0 +1,61 @@
...
+    
+    background-color: #FAD67C;
+    border-radius: 30rem 0rem 0rem 30rem;
+    width: 29.25rem;
+    padding: 0.3rem;
+    margin:1rem;
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25), -3px -4px 5px rgba(254, 226, 153, 0.3);
+    display: flex;
+    flex-direction: row;
+    justify-content: start;
```
Comment (davidlopez9926) [inline src/Pokeitem/Pokeitem.scss]: use start-flex instead

File: src/App.js [hunk 13]
```diff
@@ -1,11 +1,17 @@
...
 import './App.css';
 import Navbar from './Navbar/Navbar';
+import Pokeitem from './Pokeitem/Pokeitem'
 
 function App() {
   return (
     <div className="App">
-      <Navbar/>
+      <Navbar></Navbar>
+      <Pokeitem></Pokeitem>
```
Comment (bruno9926) [inline src/App.js]: component receives all data as props

**Review by bruno9926 — CHANGES_REQUESTED** [review]

**Review by bruno9926 — APPROVED** [review]

**Pull request merged**